merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/ResponseSerializers.java

```diff
@@ -152,7 +152,7 @@ public static JsonValue serializeSimulatedActivity(final SimulatedActivity simul
     return Json
         .createObjectBuilder()
         .add("type", simulatedActivity.type)
-        .add("parameters", serializeArgumentMap(simulatedActivity.arguments))
+        .add("arguments", serializeArgumentMap(simulatedActivity.arguments))
```

Comment:
I really appreciate that this API-facing change is separated out into its own commit 😄